schemas/data/chapter-count.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/chapter-count",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Chapter Count",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/metricdefinition"
+  ],
+  "description": "Describes the number of chapters started during the media session.",
+  "definitions": {
+    "metricdefinition": {
```

Comment:
Another set of copy/paste and incomplete editing...

Reply:
Taken out this file as it was using the legacy metricdefinition. Using measure instead.